src/css/main.css

```diff
@@ -355,3 +384,17 @@ body {
 #colorPickerContainer input:nth-child(1) {
   margin-right: 35px;
 }
+
+#stylePickerContainer {
+  display: none;
+  position: absolute;
+  left: 0;
+  top: 80px;
+}
+
+#logoPickerContainer {
+  display: none;
+  position: absolute;
+  left: 0;
+  top: 40px;
+}
```

Comment:
Use the following heroicons SVGs for the button UI.

## Logo: 
```html
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor">
  <path stroke-linecap="round" stroke-linejoin="round" d="M2.25 15.75l5.159-5.159a2.25 2.25 0 013.182 0l5.159 5.159m-1.5-1.5l1.409-1.409a2.25 2.25 0 013.182 0l2.909 2.909m-18 3.75h16.5a1.5 1.5 0 001.5-1.5V6a1.5 1.5 0 00-1.5-1.5H3.75A1.5 1.5 0 002.25 6v12a1.5 1.5 0 001.5 1.5zm10.5-11.25h.008v.008h-.008V8.25zm.375 0a.375.375 0 11-.75 0 .375.375 0 01.75 0z" />
</svg>
```
![Screenshot 2023-10-22 at 7 51 28 PM](https://github.com/p2plabsxyz/dscan/assets/68826419/23b89548-3c8d-4d8b-a089-e1ba44691ad6)

## QR code style: 
```html
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor">
  <path stroke-linecap="round" stroke-linejoin="round" d="M3.75 4.875c0-.621.504-1.125 1.125-1.125h4.5c.621 0 1.125.504 1.125 1.125v4.5c0 .621-.504 1.125-1.125 1.125h-4.5A1.125 1.125 0 013.75 9.375v-4.5zM3.75 14.625c0-.621.504-1.125 1.125-1.125h4.5c.621 0 1.125.504 1.125 1.125v4.5c0 .621-.504 1.125-1.125 1.125h-4.5a1.125 1.125 0 01-1.125-1.125v-4.5zM13.5 4.875c0-.621.504-1.125 1.125-1.125h4.5c.621 0 1.125.504 1.125 1.125v4.5c0 .621-.504 1.125-1.125 1.125h-4.5A1.125 1.125 0 0113.5 9.375v-4.5z" />
  <path stroke-linecap="round" stroke-linejoin="round" d="M6.75 6.75h.75v.75h-.75v-.75zM6.75 16.5h.75v.75h-.75v-.75zM16.5 6.75h.75v.75h-.75v-.75zM13.5 13.5h.75v.75h-.75v-.75zM13.5 19.5h.75v.75h-.75v-.75zM19.5 13.5h.75v.75h-.75v-.75zM19.5 19.5h.75v.75h-.75v-.75zM16.5 16.5h.75v.75h-.75v-.75z" />
</svg>
```
![Screenshot 2023-10-22 at 7 51 14 PM](https://github.com/p2plabsxyz/dscan/assets/68826419/0255ff13-0f11-4d4d-aa9e-9cf0a40aaadc)

## CSS:
It's pretty much same as the other SVG buttons, use this:
```css
logo and style {
  background-color: white;
  width: 30px;
  height: 30px;
  padding: 6px;
  border: 2px solid #c3c0c0;
  color: #326d6e;
  border-radius: 5px;
  text-align: center;
  align-items: center;
  -webkit-box-shadow: 0 2px 6px -6px #222;
  -moz-box-shadow: 0 2px 6px -6px #222;
  box-shadow: 0 2px 6px -6px #222;
}

logo and style:hover {
  color: #175859;
  border-color: #cccdcd;
  cursor: pointer;
}
```

For the alignment and positioning of the buttons, look at this picture and align it accordingly:
![274641757-219eae88-4ce8-4c1d-b8ce-b89bb0a0570c](https://github.com/p2plabsxyz/dscan/assets/68826419/5155d3c8-0e56-4c03-b6de-e309851e4478)